tests/test_top.py

```diff
@@ -403,14 +405,15 @@ def test_top(read_delay,
     wen_en = 1
     ren_en = [0] * interconnect_output_ports
     addr_in = 0
-
+    output_en = 1
```

Comment:
move 416 up here since this is written over

Reply:
I inited to 0